src/monitoring_service/blockchain.py

```diff
@@ -100,159 +100,177 @@ def query_blockchain_events(
-            to_block=to_block,
-        )
-
-    def _get_token_networks_events(
-        self, network_address: TokenNetworkAddress, from_block: BlockNumber, to_block: BlockNumber
-    ) -> List[Dict]:
-        return query_blockchain_events(
-            web3=self.w3,
-            contract_manager=self.contract_manager,
-            contract_address=Address(network_address),
-            contract_name=CONTRACT_TOKEN_NETWORK,
-            topics=[None],
-            from_block=from_block,
-            to_block=to_block,
-        )
-
-    def _get_monitoring_service_events(
-        self, monitoring_service_address: Address, from_block: BlockNumber, to_block: BlockNumber
-    ) -> List[Dict]:
-        return query_blockchain_events(
-            web3=self.w3,
-            contract_manager=self.contract_manager,
-            contract_address=monitoring_service_address,
-            contract_name=CONTRACT_MONITORING_SERVICE,
-            topics=[None],
-            from_block=from_block,
-            to_block=to_block,
-        )
+def _get_token_network_registry_events(
+    web3: Web3,
+    contract_manager: ContractManager,
+    registry_address: Address,
+    from_block: BlockNumber,
+    to_block: BlockNumber,
+) -> List[Dict]:
+    return query_blockchain_events(
+        web3=web3,
+        contract_manager=contract_manager,
+        contract_address=registry_address,
+        contract_name=CONTRACT_TOKEN_NETWORK_REGISTRY,
+        topics=create_registry_event_topics(contract_manager),
+        from_block=from_block,
+        to_block=to_block,
+    )
 
-    def get_events(
-        self, chain_state: BlockchainState, to_block: BlockNumber
-    ) -> Tuple[BlockchainState, List[Event]]:
-        # increment by one, as latest_known_block has been queried last time already
-        from_block = BlockNumber(chain_state.latest_known_block + 1)
 
-        # Check if the current block was already processed
-        if from_block > to_block:
-            return chain_state, []
+def _get_token_networks_events(
+    web3: Web3,
+    contract_manager: ContractManager,
+    network_address: TokenNetworkAddress,
+    from_block: BlockNumber,
+    to_block: BlockNumber,
+) -> List[Dict]:
+    return query_blockchain_events(
+        web3=web3,
+        contract_manager=contract_manager,
+        contract_address=Address(network_address),
+        contract_name=CONTRACT_TOKEN_NETWORK,
+        topics=[None],
+        from_block=from_block,
+        to_block=to_block,
+    )
 
-        new_chain_state = deepcopy(chain_state)
-        log.info('Querying new block(s)', from_block=from_block, end_block=to_block)
 
-        # first check for new token networks and add to state
-        registry_events = self._get_token_network_registry_events(
-            registry_address=new_chain_state.token_network_registry_address,
-            from_block=from_block,
-            to_block=to_block,
-        )
+def _get_monitoring_service_events(
```

Comment:
Is it worth it to have these three `_get_..._events` functions? Their signature is about as long as their implementation.